packages/cosmic-proto/test/address-hooks.test.js

```diff
@@ -0,0 +1,214 @@
+import rawTest from '@endo/ses-ava/prepare-endo.js';
+
+import bundleSourceAmbient from '@endo/bundle-source';
+import { importBundle } from '@endo/import-bundle';
+
+/**
+ * @type {import('ava').TestFn<{
+ *   addressHooks: import('../src/address-hooks.js');
+ * }>}
+ */
+const test = rawTest;
+
+const makeTestContext = async () => {
+  const bundleSource = bundleSourceAmbient;
+  const loadBundle = async specifier => {
+    const modulePath = new URL(specifier, import.meta.url).pathname;
```

Comment:
I have a vague memory that this `URL(...).pathname` idiom is to be avoided.

@kriskowal , does this ring a bell?

My habit is to use `createRequire`. In fact, i'm pretty sure I picked that up from you, @michaelfig .